src/main/java/fr/minuskube/inv/content/InventoryContents.java

```diff
@@ -163,6 +164,23 @@ public InventoryContents add(ClickableItem item) {
             return this;
         }
 
+        /**
+         * Updates an item in this inventory without reloading the entire GUI.
+         * @param slotPos the position of the item to update
+         * @param itemStack the new {@code ItemStack} that will replace the old one
+         * @return this {@code InventoryContents} instance
+         */
```

Comment:
actually, javadocs should be in the interfaces, not implementations.

Reply:
My bad. I totally forgot about that. I moved the JavaDoc in the right place.